src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -323,29 +323,20 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('doctrine.xml');
     }
 
-    /**
-     * @param $webspaceConfig
-     */
-    private function initWebspace($webspaceConfig, ContainerBuilder $container, Loader\XmlFileLoader $loader)
+    private function initWebspace(array $webspaceConfig, ContainerBuilder $container, XmlFileLoader $loader)
     {
         $container->setParameter('sulu_core.webspace.config_dir', $webspaceConfig['config_dir']);
 
         $loader->load('webspace.xml');
     }
 
-    /**
-     * @param $fieldsConfig
-     */
-    private function initFields($fieldsConfig, ContainerBuilder $container)
+    private function initFields(array $fieldsConfig, ContainerBuilder $container)
     {
         $container->setParameter('sulu.fields_defaults.translations', $fieldsConfig['translations']);
         $container->setParameter('sulu.fields_defaults.widths', $fieldsConfig['widths']);
     }
 
-    /**
-     * @param $contentConfig
-     */
-    private function initContent($contentConfig, ContainerBuilder $container, Loader\XmlFileLoader $loader)
+    private function initContent(array $contentConfig, ContainerBuilder $container, XmlFileLoader $loader)
```

Comment:
we cannot add a `array` typehint, we need to use a phpdoc comment

Reply:
We did it but the question is do we need to do it on a private function?

Reply:
no, you are right. on private functions we can add typehints. i did not see that this is a private function 🙈

Reply:
i added the typehint again and will merge as soon as the ci is green 🙂